nilearn/input_data/tests/test_nifti_spheres_masker.py

```diff
@@ -70,3 +70,25 @@ def test_anisotropic_sphere_extraction():
 def test_errors():
     masker = NiftiSpheresMasker(([1, 2]), radius=.2)
     assert_raises_regex(ValueError, 'Seeds must be a list .+', masker.fit)
+
+
+def test_nifti_spheres_masker_overlap():
+    # Test resampling in NiftiMapsMasker
+    affine = np.eye(4)
+    shape = (5, 5, 5)
+
+    data = np.random.random(shape + (5,))
+    fmri_img = nibabel.Nifti1Image(data, affine)
+
+    seeds = [(0, 0, 0), (2, 2, 2)]
+
+    o_masker = NiftiSpheresMasker(seeds, radius=1, allow_overlap=True)
```

Comment:
Original remark about o -> overlapping, no -> non_overlapping applied to this file as well.